appveyor.yml

```diff
@@ -67,11 +77,10 @@ build_script:
       -D ENABLE_BENCHMARKS=%ENABLE_BENCHMARKS% ^
       -D ENABLE_OPENMP=ON ^
       -D ENABLE_MPI=%ENABLE_MPI% ^
-      -D MPI_HOME:PATH="C:/Program Files (x86)/Microsoft SDKs/MPI" ^
-      -D MPI_C_INCLUDE_PATH:PATH="${MPI_HOME}/Include" ^
-      -D MPI_C_LIBRARIES:PATH="${MPI_HOME}/Lib/x86/msmpi.lib" ^
-      -D MPI_CXX_INCLUDE_PATH:PATH="${MPI_HOME}/Include" ^
-      -D MPI_CXX_LIBRARIES:PATH="${MPI_HOME}/Lib/x86/msmpi.lib" ^
+      -D MPI_C_INCLUDE_PATH:PATH="%MPI_HOME%/Include" ^
+      -D MPI_C_LIBRARIES:PATH="%MPI_HOME%/Lib/x86/msmpi.lib" ^
```

Comment:
Not sure if this is the problem here, but I've been able to get around an improper ``PATH`` issue in the path by telling cmake that it was a ``STRING``.   E.g. ``-D MPI_C_INCLUDE_PATH:STRING="%MPI_HOME%/Include``

Reply:
not a powershell pro, but i think for this style, the string will be sub-ed in before passed to cmake. The last style wasn't doing this & while wasn't a problem for cmake 3.8 and 3.9, it actually did cause an issue for 3.10.  That was the only thing I solved for 3.10 before climbing out of the rabbit hole.